packages/editor/src/components/block-mover/arrows.js

```diff
@@ -14,3 +14,11 @@ export const downArrow = (
 		<polygon points="9,13.5 14.7,7.9 13.2,6.5 9,10.7 4.8,6.5 3.3,7.9 " />
 	</AccessibleSVG>
 );
+
```

Comment:
Should we rename this file `icons` instead of `arrows`?

Reply:
e9dc8aae6a01f34b01c771105ff024600402cd08